server/src/models/barrier.ts

```diff
@@ -0,0 +1,18 @@
+import { Table, Model, Column, DataType, BelongsTo, ForeignKey } from "sequelize-typescript"
+import { Map } from "./map"
+
+@Table({ tableName: "Barriers", })
+export class Barrier extends Model<Barrier> {
+  @BelongsTo(() => Map)
+    map!: Map
+
+  @ForeignKey(() => Map)
+  @Column
+    mapId!: number
+
+  @Column(DataType.INTEGER)
+    x!: number
+
+  @Column(DataType.INTEGER)
+    y!: number
```

Comment:
Do you think we should make these column names more descriptive like your naming convention in `map.ts` with `barrier_x` or maybe even `barrier_x_cord` (but we'd need to update other records using similar coordinates)?